dsmr_parser/telegram_specifications.py

```diff
@@ -1233,7 +1132,7 @@
             # This seems to be wrong in documentation, it's not 0-0:96.50.68, but 0-0:96.3.10
         },
         {
-            'obis_reference': obis.BELGIUM_MAX_POWER_PER_PHASE,
```

Comment:
Seem unrelated to MBUS, can this be moved to another PR?

Reply:
See above :)